src/pages/store/Store.tsx

```diff
@@ -0,0 +1,73 @@
+import CokoLogo from '@common/layout/CokoLogo';
+import Header from '@common/layout/Header';
+import MenuBar from '@common/layout/MenuBar';
+import * as globalS from '@/style/style';
+import * as S from './styles';
+import { useState } from 'react';
+import ItemContainer from '@features/store/ui/ItemContainer';
+import CartList from '@features/store/ui/CartList';
+import MyCharacter from '@features/user/ui/MyCharacter';
+import ProfileImage from '@features/user/ui/ProfileImage';
+import Item from '@type/Item';
+const buttonList: { label: string; name: Item['category'] }[] = [
+  {
+    label: '의상',
+    name: 'clothes',
+  },
+  {
+    label: '악세사리',
+    name: 'accessories',
+  },
+  {
+    label: '프로필',
+    name: 'profile',
+  },
+  {
+    label: '색상',
+    name: 'color',
+  },
+];
```

Comment:
```suggestion
const buttonList = [
  {
    label: '의상',
    name: 'clothes',
  },
  {
    label: '악세사리',
    name: 'accessories',
  },
  {
    label: '프로필',
    name: 'profile',
  },
  {
    label: '색상',
    name: 'color',
  },
] as const;
```
이런 배열, 객체의 경우 `as const`로 관리하는게 더 명확하게 관리 할 수 있을 것 같은데 혹시 이슈가 있던걸까요?

Reply:
https://github.com/modern-agile-team/8term-coko-Front/pull/84#discussion_r1874880081
해당 코멘트에 남겨놓았습니다!

Reply:
이전에 Enum타입을 찾아보다 대체제로 `as const`라는 걸 처음 알게 되었는데 이런 식으로 쓰일 수 있군요! 감사합니다~

[[Typescript] as const에 대하여](https://velog.io/@taewo/Typescript-as-const%EC%97%90-%EB%8C%80%ED%95%98%EC%97%AC)